pages/common/for.md

```diff
@@ -0,0 +1,11 @@
+# for
+
+> Shell loop over parameters
+
+- Perform an action with different arguments.
```

Comment:
Why use "argument" in the command description but "parameter" in the actual command? I'd suggest unifying the nomenclature.